### DISCUSSION TITLE
How do I keep the height and width in a 4/3 ratio?

### DISCUSSION THREAD

**Discussion by faruuhan** [mantinedev/mantine#1481] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/92959224/169019946-44af0d73-d534-40ca-89dd-5773ceb4a322.png)

```
<AspectRatio ratio={4 / 3} style={{ width: "200px" }}>
       <Image src={items.urlToImage} alt={items.title} withPlaceholder />
</AspectRatio>
```

**Comment by rtivital** [comment, marked as answer]
AspectRatio will not work well in this case – Image wraps img tag with additional elements, you should customize Image with styles API instead or use img tag.
